src/Attestation/Attestation/Commands/GetAzureAttestation.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.Azure.Commands.Attestation.Models;
+using Microsoft.Azure.Commands.Attestation.Properties;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Collections;
+using System.Management.Automation;
+using System.Runtime.InteropServices.ComTypes;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Rest.Azure;
+
+namespace Microsoft.Azure.Commands.Attestation
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Attestation", SupportsShouldProcess = true, DefaultParameterSetName = NameParameterSet)]
```

Comment:
@shleiAmy if this is a tracked resource (_i.e._, has a resource id), we should add a parameter set that allows the user to provide or pipe a resource id, via a `-ResourceId` parameter, to this cmdlet that we parse and pull the resource group name and resource name from.

More information about this scenario can be found [here](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/piping-best-practices.md#using-the--resourceid-parameter).

Reply:
ResourceGroupParameterSet is added